Data/Graph/Inductive/Query/Cycles.hs

```diff
@@ -0,0 +1,280 @@
+{- |
+   Module      : Data.Graph.Inductive.Query.Cycles
+   Description : Algorithms for finding all cycles.
+   Copyright   : (c) Gabriel Hjort Blindell 2017
+   License     : 2-Clause BSD
+   Maintainer  : gabriel.hjort.blindell@gmail.com
```

Comment:
Will you be maintaining this module?

(Actually, if you look at all the current FGL modules they tend not to bother with the header block at all, so you could just remove the entire thing.)